src/common/MoviesList/index.js

```diff
@@ -1,58 +1,36 @@
 import React from "react";
```

Comment:
I think this import is not necessary ;-)

Reply:
Import removed, thanks for the feedback! 💪